src/gc/gc.cpp

```diff
@@ -1892,29 +1883,22 @@ static void leave_spin_lock (GCSpinLock * spin_lock)
 
 #endif //_DEBUG
 
-BOOL gc_heap::enable_preemptive (Thread* current_thread)
+BOOL gc_heap::enable_preemptive ()
 {
-    bool cooperative_mode = false;
-    if (current_thread)
+    bool cooperative_mode = GCToEEInterface::IsPreemptiveGCDisabled();
+    if (cooperative_mode)
     {
-        cooperative_mode = GCToEEInterface::IsPreemptiveGCDisabled(current_thread);
-        if (cooperative_mode)
-        {
-            GCToEEInterface::EnablePreemptiveGC(current_thread);
-        }
+        GCToEEInterface::EnablePreemptiveGC();
     }
 
     return cooperative_mode;
 }
 
-void gc_heap::disable_preemptive (Thread* current_thread, BOOL restore_cooperative)
+void gc_heap::disable_preemptive (BOOL restore_cooperative)
```

Comment:
It would be nice to chance these to lower case `bool` while you are changing the code.

Reply:
ok!